_includes/disqus_comments.html

```diff
@@ -0,0 +1,20 @@
+{% if page.comments != false and jekyll.environment == "production" %}
+
```

Comment:
I'm not sure if this is just a default template or if this means that we are supporting comments, but we may want to look into ensuring that comments are disabled until the team has a discussion regarding support and moderation.